plugins/notify/__init__.py

```diff
@@ -333,14 +333,14 @@ def __init_notifier(self, caps):
         return GLib.SOURCE_REMOVE
 
     def disable(self, _exaile):
-        if self.__notifier:
-            self.__notifier.destroy()
-            self.__notifier = None
+        self.teardown(exaile)
```

Comment:
Must change the methods signature, probably will fail because "exaile" isn't defined (not tested).

Reply:
Ooops, you're right. I missed that underscore in method's signature.